### PR TITLE
LS models: Update to new ls:create-*models

### LINKED ISSUE #294: Rename LevelSpace "Model Loader Example"?
Given https://github.com/NetLogo/LevelSpace/issues/84, **Model Loader Example** may not make sense anymore as the name for that code example.

@qiemem has suggested **Parent Model Example**, which may not be a bad idea.

In any case, if the model is renamed:

- The references to that model in the info tabs of other examples need to be updated:
  - [ ] **Model Interactions Example**
  - [ ] **Model Visualizer and Plotter Example**
- [ ] `Model Loader Example.png` needs to be renamed too.

### PR DESCRIPTION
**Pull request by qiemem**
Fixes #294

### CODE REVIEW

**Comment by bainco** [comment]
Tests won't pass until new LS makes it into hexy. Or, if we want, we can point tests manually.

File: Curricular Models/GenEvo/GenEvo 4 Competition.nlogo [hunk 1]
```diff
@@ -48,28 +48,28 @@ to generate-cells
     set shape "ecoli"
 
     ifelse choose-models? [
-      user-message "Select a Genetic Switch model to load..."
+      user-message "Select a Genetic Switch model to create..."
```
Comment (nicolaspayette) [inline Curricular Models/GenEvo/GenEvo 4 Competition.nlogo]: @qiemem, @bainco: Not sure what to suggest instead, but that turn of phrase is awkward. Maybe "Select a Genetic Switch model to use as a child model..."? Or "from which to create a child model"? They're both a mouthful, though.

Comment (qiemem) [inline Curricular Models/GenEvo/GenEvo 4 Competition.nlogo]: Alternatively: "Select a Genetic Switch model to use."

Comment (nicolaspayette) [inline Curricular Models/GenEvo/GenEvo 4 Competition.nlogo]: Ya, I'm OK with that.

Comment (bainco) [inline Curricular Models/GenEvo/GenEvo 4 Competition.nlogo]: Me too! @nicolaspayette, are you pushing a change?

File: Curricular Models/GenEvo/GenEvo 4 Competition.nlogo [hunk 2]
```diff
@@ -10,7 +10,7 @@ breed [ ecolis ecoli ]
 
 ecolis-own [
   my-model              ; cell model (LevelSpace child model) associated with an E. coli cell
-  my-model-path         ; path to locate the file to be loaded as a cell model (LevelSpace child model)
+  my-model-path         ; path to locate the file to be created as a cell model (LevelSpace child model)
```
Comment (nicolaspayette) [inline Curricular Models/GenEvo/GenEvo 4 Competition.nlogo]: "the file to be created": that is also weird. I'll replace it with "the file to be used for creating".

File: Curricular Models/GenEvo/GenEvo 4 Competition.nlogo [hunk 3]
```diff
@@ -389,9 +389,9 @@ Because the molecular interactions in each cell model are stochastic, the popula
 
 You can use this model in two ways.
 
-1. The simplest way to use this model is to set CHOOSE-MODELS? to OFF. In this mode, the default Genetic Switch model is loaded for each cell. This method simulates genetic drift as a mechanism of evolution since each cell starts off identical.
+1. The simplest way to use this model is to set CHOOSE-MODELS? to OFF. In this mode, the default Genetic Switch model is created for each cell. This method simulates genetic drift as a mechanism of evolution since each cell starts off identical.
 
-2. If CHOOSE-MODELS? is ON, a user can manually select the NetLogo models (`.nlogo` files) to be loaded for each cell, allowing each cell to have a different genetic switch. This allows for variability in the population. This method simulates natural selection as well as genetic drift.
+2. If CHOOSE-MODELS? is ON, a user can manually select the NetLogo models (`.nlogo` files) to be created for each cell, allowing each cell to have a different genetic switch. This allows for variability in the population. This method simulates natural selection as well as genetic drift.
```
Comment (nicolaspayette) [inline Curricular Models/GenEvo/GenEvo 4 Competition.nlogo]: I will adjust this one too. We need to be careful to not make it sound as if it is the file that is created.

**Pull request merged**